<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_gerenciarEstoque</class>
 <widget class="QDialog" name="window_gerenciarEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciar Estoque</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <widget class="QTabWidget" name="tb_gerenciarProdutos">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_novoProduto">
      <attribute name="title">
       <string>Novo Produto</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>530</width>
         <height>281</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Código</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_codigoProduto"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Descrição</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_descricaoProduto"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Fornecedor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_fornecedor"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Valor Custo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_valorCusto"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Quantidade Estoque</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_qtdEstoque"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Valor Venda</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_valorVenda"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btn_limpar">
              <property name="text">
               <string>Limpar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_gravarProduto">
              <property name="text">
               <string>Gravar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_gerenciarProduto">
      <attribute name="title">
       <string>Gerenciar Produtos</string>
      </attribute>
      <widget class="QTableWidget" name="tw_geProdutos">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>121</y>
         <width>371</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>20</y>
         <width>120</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Filtro</string>
       </property>
       <widget class="QRadioButton" name="rb_geCodigo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>100</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Código</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_geDescricao">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>100</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Descrição</string>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="txt_gePesquisa">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>80</y>
         <width>191</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>40</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pesquisa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_geBuscar">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>40</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>321</width>
         <height>411</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Código</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_geProduto"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_geDescricaoProduto"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Quantidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_geQuantidade"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Valor Custo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_geValorCusto"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Valor Venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_geValorVenda"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_geFornecedor"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="btn_geGravar">
            <property name="text">
             <string>Gravar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_geExcluir">
            <property name="text">
             <string>Excluir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txt_codigoProduto</tabstop>
  <tabstop>txt_descricaoProduto</tabstop>
  <tabstop>txt_fornecedor</tabstop>
  <tabstop>txt_valorCusto</tabstop>
  <tabstop>txt_qtdEstoque</tabstop>
  <tabstop>txt_valorVenda</tabstop>
  <tabstop>tb_gerenciarProdutos</tabstop>
  <tabstop>btn_gravarProduto</tabstop>
  <tabstop>btn_limpar</tabstop>
  <tabstop>txt_geProduto</tabstop>
  <tabstop>txt_geDescricaoProduto</tabstop>
  <tabstop>txt_geValorVenda</tabstop>
  <tabstop>txt_geQuantidade</tabstop>
  <tabstop>txt_geFornecedor</tabstop>
  <tabstop>txt_geValorCusto</tabstop>
  <tabstop>btn_geGravar</tabstop>
  <tabstop>btn_geExcluir</tabstop>
  <tabstop>tw_geProdutos</tabstop>
  <tabstop>rb_geCodigo</tabstop>
  <tabstop>rb_geDescricao</tabstop>
  <tabstop>txt_gePesquisa</tabstop>
  <tabstop>btn_geBuscar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
